### PR TITLE
Create ui-c8y-1020-0-0-change-docker-image-version.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by dmaj

Ticket: [DM-3650](https://cumulocity.atlassian.net/browse/DM-3650)
Original PR: [6240](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6240)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-change-docker-image-version.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Change docker image version
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-change-docker-image-version.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Update Docker image to latest version
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-change-docker-image-version.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3650
+version: 1020.0.0
+---
+Change docker image version
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-change-docker-image-version.md:17]: The Claude AI suggested to use the following body:
```suggestion
As part of our continuous improvement efforts, we have updated the Docker image to the latest version. The new Docker image includes performance enhancements, bug fixes, and security updates. This change ensures that your deployments using our Docker image will benefit from the latest improvements and have access to the most recent features. The update is fully backward compatible and should not require any modifications to your existing deployment configuration.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by reey** [comment]
closing as only internal

**Pull request closed without merging**